XCUITests/FxScreenGraph.swift

```diff
@@ -332,4 +333,36 @@ extension Navigator {
             self.goto(TabTray)
         }
     }
+
+    func selectOptionFromBrowserTabMenu(view: selectMenuOption) {
```

Comment:
Option 1 is better than option 2. However, I think the quick and dirty solution would be to do something like 

```swift
let page1Options = [ requestDesktop, requestMobile, findInPageOption, 
                                   requestSetHomePage, addBookmarkOption, removeBookmarkOption ]
if page1Options.contains(view) {
   // take the option
} else {
   goto(BrowserTabMenu2)
   // take the option
}
```